internal/pkg/ecslogging/service.go

```diff
@@ -53,12 +52,15 @@ func NewServiceClient(sess *session.Session, app, env, svc string) *ServiceClien
 
 // WriteLogEvents writes service logs.
 func (s *ServiceClient) WriteLogEvents(opts WriteLogEventsOpts) error {
-	logEventsOutput := &cloudwatchlogs.LogEventsOutput{
-		LastEventTime: make(map[string]int64),
+	logEventsOpts := &cloudwatchlogs.LogEventsOpts{
+		LogGroup:  s.logGroupName,
+		Limit:     aws.Int64(int64(opts.Limit)),
+		EndTime:   opts.EndTime,
+		StartTime: opts.StartTime,
+		// TODO: Add log filtering by task ID.
 	}
-	var err error
 	for {
-		logEventsOutput, err = s.eventsGetter.LogEvents(s.logGroupName, logEventsOutput.LastEventTime, opts.generateGetLogEventOpts()...)
+		logEventsOutput, err := s.eventsGetter.LogEvents(logEventsOpts)
```

Comment:
nit: Should we be consistent with the type of the inputs? `LogEvents` takes a pointer vs `WriteLogEvents` that takes a value. 

I think in this situation, since we don't modify the given input we should also change `LogEvents` to be a value.